rplidar_test_utils.go

```diff
@@ -0,0 +1,74 @@
+//nolint:golint
+package rplidar
+
+import (
+	"go.viam.com/rplidar/inject"
+
+	"go.viam.com/rplidar/gen"
+)
+
+// GoodRplidarReturnsZeroPoints creates a Rplidar that returns only zero data
+func GoodRplidarReturnsZeroPoints() *rplidar {
```

Comment:
[note to self] so the setup for these tests is to have each table test pass in an injected rplidar, which will be defined in this file

Reply:
File has been removed as we are not going to be testing success cases yet